Wendy.jiang/update event management doc (#22900)
DataDog/documentation: content/en/service_management/events/correlation/patterns.md

{{< img src="service_management/events/correlation/pattern/recommended_patterns_preview.png" alt="Correlation recommended patterns with the preview panel showing potential cases the pattern would create" style="width:100%;" >}}

## Create a pattern

To create a pattern:
1. Navigate to [Correlation][1].

> 1. Click **+ Add a Pattern**, at the top of the Pattern table. This opens pattern configuration page with out-of-the-box suggested patterns on the left side , and pattern output preview on the right side. 

Reviewer: ```suggestion
1. Click **+ Add a Pattern**, at the top of the Pattern table. This opens a pattern configuration page that displays out-of-the-box suggested patterns on the left side, and a pattern output preview on the right side. 
```

> 1. You can adjust the suggested pattern by clicking **+ Continue With Pattern**(this will take you to the pre-populated configuration page for additonal tuning) or choose to create your own by clicking **+ Personalize From Scratch**

Reviewer: ```suggestion
1. You can adjust a suggested pattern by clicking **+ Continue With Pattern**. This takes you to the pre-populated configuration page for additional tuning. Or, you can choose to create your own by clicking **+ Personalize From Scratch**.
```

> Events first get deduplicated to alerts based on event aggregation key, alerts get correlated to case based on configuration 

Reviewer: ```suggestion
First, events are deduplicated to alert based on event aggregation key. Then, alerts are correlated to a case based on configuration.
```

> {{< img src="service_management/events/correlation/correlation_helper.mp4" alt="Dedupe and correlate events" video=true >}}

Reviewer: Best practice is not to have text information on videos for accessibility. Also, if possible, recommend using deduplicated, instead of de-deplicated.

```suggestion
{{< img src="service_management/events/correlation/correlation_helper.mp4" alt="When a pattern matches defined alerting sources, a case is created. Source alerts are added, and its events are de-duplicated for 1 hour before the process repeats in a new case. You can modify these in advanced settings" video=true >}}
```

> Check out API or integration page about how to setup aggregation key. Events without aggregation key will be deduped to one single alert within the timeframe

Reviewer: Which API page and integration page? We should add a link here.

```suggestion
For more information on how to setup the aggregation key, see the API or integration page. Events without an aggregation key are deduped to one single alert within the timeframe.
```

> ### Suggested pattern

Reviewer: ```suggestion
### Suggested patterns
```

> Suggested pattern are recommended based on your commonly used service, environment tags to help you get started with event correlation quickly. 

Reviewer: ```suggestion
Suggested patterns are recommended based on your commonly used service and environment tags to help you get started with event correlation quickly. 
```

> ### Configuration
> Once you are on the [correlation configuration page][2]
> 1. Select the event source you want to group on from the dropdown
> 1. Define the grouping tags. Grouping tags are event facets. see advanced settings section below if you don't find the tag from the dropdown 
> 1. To exclude any events from the source defined above, adding event query in **Filter by these events or tags** to filter them out

Reviewer: ```suggestion
### Configuration
From the [correlation configuration page][2]:
1. Select the event source to group on from the dropdown.
1. Define the grouping tags. Grouping tags are event facets. See the [advanced settings section](#advanced-settings-optional) below if you don't see the tag from the dropdown. 
1. To exclude any events from the source defined above, add an event query in **Filter by these events or tags** to filter them out.
```

   : is any event that is available in your Event Explorer as an event, but does not have an associated monitor or alert.
   
   Add grouping tags

>    : to add new grouping tags, this is same as adding new event facet. 

Reviewer: Can we link to the docs on new event facets? Is this the correct [page](https://docs-staging.datadoghq.com/wendy.jiang/update-event-management-doc/service_management/events/explorer/facets/#create-a-facet)? 

```suggestion
   : to add new grouping tags, this is same as adding new event facet. 
```

    : The max duration that net new alerts will be added to a case 

    Deduplicate events for those alerts for

>     : The max duration for current alerts which have been correlated, but continue to flap or have not resolved, will be deduped to the alert in the existing case before opening new case

Reviewer: I'm not sure if my suggestion makes sense here. Just trying to break this sentence up.

```suggestion
    : The max duration for current alerts which have been correlated, but continue to flap or have not resolved. Alerts are deduped to the first alert in the existing case before opening a new case
```

## Preview pattern output

> Preview the possible patterns and cases your configuration would potentially create. The preview panel displays the total number of ingested events(limit to the first 1000 events), how many alerts would be deduped from events, and how many cases would be created based on the configuration. 

Reviewer: ```suggestion
Preview the possible patterns and cases your configuration would potentially create. The preview panel displays 
- The total number of ingested events(limited to the first 1000 events).
- The number of alerts that would be deduped from events.
- The number of cases that would be created based on the configuration. 
```

Use this data to preview the impact of your correlations and understand the expected output of a pattern.

{{< img src="service_management/events/correlation/pattern/preview_pattern_output.png" alt="Configuration for pattern-based correlation highlighting the preview panel; panel shows the number of ingested events that match your configuration, how many of those events alert, how much deduplication would occur, and the number of cases that would result." style="width:100%;" >}}

> *notes*: the title in preview case is default to the first alert in correlation. in real event management case, it will be intelligently generated. 

Reviewer: ```suggestion
**Note**: the default title in the preview case is the first alert in correlation. After you save a pattern, the event management case title is intelligently generated. 
```

## Select a Case Management destination

1. From the *Project* dropdown menu, select from an existing Case to send your grouped events to.
1. (Optional) Add a tag to resulting cases.
1. Click **Save and Activate** to activate this pattern and group events into cases.

## Update existing pattern

> *notes*: Once you update existing pattern, all current live cases will stop processing. New event matches the pattern will create new case. 

Reviewer: ```suggestion
After you update an existing pattern, all live cases will stop processing. New events that match the pattern will create a case. 
```
